modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -71,7 +91,21 @@ curl --json '{"model_id": "<model_name>", "inputs": "<text>"}' \
 https://<inference_endpoint_url>:443/api/v1/task/server-streaming-text-generation \
 -H 'Authorization: Bearer <token>' <1>
 ----
-<1> You must add the `Authorization` header and specify a token value _only_ if you enabled token authorization when deploying the model.
+<1> Add the `Authorization` header and specify a token value only if you enabled token authorization when deploying the model.
+
+*Caikit Standalone ServingRuntime for KServe*
+
+.HTTP
+[source]
+----
+curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' /api/v1/task/embedding
```

Comment:
This query doesn't look quite right to me. I think it should include the inference endpoint and port number before the `/api`... path, like line https://github.com/opendatahub-io/opendatahub-documentation/pull/343/files#diff-c2e886ee656c6229e397ac0c53b02db917ab03100d8ff1cdaeb8389b8423f1d5R158.

Reply:
Yes, it should be: 
```
curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' <inference_endpoint_url>/api/v1/task/embedding
```

Reply:
@christinaexyou Just want to double-check something:

I don't think the port number is part of the inference endpoint shown in the dashboard?

Therefore, the query would need to be this:

```
curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' <inference_endpoint_url>:8080/api/v1/task/embedding -H 'Authorization: Bearer <token>' <1>
```

WDYT?

Reply:
@jbyrne-redhat there's 2 ways of making REST requests:

1. With the model route:
```
curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' <inference_endpoint_url>/api/v1/task/embedding -H 'Authorization: Bearer <token>'
```

2. With the host and port
```
curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' localhost:8080/api/v1/task/embedding -H 'Authorization: Bearer <token>'
```

So you would only need to include the port if you are querying the host.